### DISCUSSION TITLE
Playlist navigation

### DISCUSSION THREAD

**Discussion by HelpSeeker** [Rafostar/clapper#62] [Q&A] [answered]
Is it yet possible to navigate to the next/previous playlist item via keybinds? And if yes, what keys are currently used for it?

**Comment by Rafostar** [comment, marked as answer]
Not at the moment unfortunately. I need to finish what I have right now and release it as 0.2.0 ver soon. The plan is to have more keybindings and a window that shows all these shortcuts for 0.3.0 ver a little later on.

Please open this as a feature request in issues, so it will be easier to track progress for it.

**Reply by HelpSeeker** [reply]
Thanks for the answer. I understand that a relatively new project with a such a big scope takes time to develop. I'll look forward to future versions and open this as a feature request in the meantime.
